### DISCUSSION TITLE
How to print the huge organic layout structure content in the page using part by part

### DISCUSSION THREAD

**Discussion by venkatN467** [maxGraph/maxGraph#519] [Q&A] [answered]
is there any way to print the huge organic layout structure content in the page using part by part. (we have to sniff off part by part and add in excel one by one).

**Comment by tbouffard** [comment]
Hi, sorry I don't understand

Can you provide about what you are exactly trying to achieve, what the current problem is, and how you currently trying to workaround it?

**Reply by venkatN467** [reply]
Hi,
Thanks for your reply, please find the below screenshot. i am trying to check is it possible to print all the alphabets and relation between them into a excel sheet.
![OrganicLayout](https://github.com/user-attachments/assets/559b67c6-10c1-4cc9-b3a4-68a3636522a6)

**Reply by tbouffard** [reply, marked as answer]
This should work as in mxGraph

mxGraph example:
- [Orgchart](https://jgraph.github.io/mxgraph/javascript/examples/orgchart.html): Using automatic layouts, fit to page zoom and poster print (across multiple pages).
- [Pagebreaks](https://jgraph.github.io/mxgraph/javascript/examples/pagebreaks.html) - Using the pageBreaksVisible and preferPageSize switches and adding headers and footers to print output.

The corresponding maxGraph examples are
- https://maxgraph.github.io/maxGraph/demo/?path=/story/layouts-orgchart--default: this example has not be correctly migrated, it is not working. This doesn't means that the underlying maxGraph code doesn't work
- https://maxgraph.github.io/maxGraph/demo/?path=/story/printing-pagebreaks--default
